### DISCUSSION TITLE
Does Rollouts work on k8s 1.27?

### DISCUSSION THREAD

**Discussion by PDVJAM** [argoproj/argo-rollouts#2987] [Q&A]
Does Rollouts work on k8s 1.27?

**Comment by kaiburjack** [comment]
Yes, it does. We have been using it for quite some weeks now without problems. There are no Kubernetes Resources in the Argo Rollouts Helm Charts that won't work on 1.27.

**Reply by Hesham-XQ** [reply]
which version of argo rollouts is better compiatible with 1.27 ??
